install.sh

```diff
@@ -967,11 +967,11 @@ service_enable_and_start() {
         return
     fi
 
-    if command -v iptables-save &> /dev/null && command -v iptables-restore &> /dev/null
+    if command -v iptables-save && command -v iptables-restore
     then
 	    $SUDO iptables-save | grep -v KUBE- | grep -v CNI- | grep -iv flannel | $SUDO iptables-restore
     fi
-    if command -v ip6tables-save &> /dev/null && command -v ip6tables-restore &> /dev/null
+    if command -v ip6tables-save && command -v ip6tables-restore
```

Comment:
Won't this cause the path to the various commands to be printed to the terminal whenever the install script is run? I'd prefer to fix the shell syntax so that the redirection works properly.

Reply:
The issue I think it's related only to ubuntu and the `&>` redirect with other OS the output is rightly redirected but on Ubuntu it's not and the if condition is always true I tried with `1>` and it seems to work also on ubuntu.

Reply:
It's because the ubuntu minimal `sh` is actually dash :(

```console
ubuntu@ip-172-31-5-176:~$ ls -la /usr/bin/sh
lrwxrwxrwx 1 root root 4 Mar 23  2022 /usr/bin/sh -> dash

ubuntu@ip-172-31-5-176:~$ bash -c 'command -v iptables-save &> /dev/null && echo foo'

ubuntu@ip-172-31-5-176:~$ sh -c 'command -v iptables-save &> /dev/null && echo foo'
foo

ubuntu@ip-172-31-5-176:~$ sh -c 'command -v iptables-save 1> /dev/null && echo foo'
```

This is stupid and I hate it, but clearly something we'll need to work around.

Reply:
My bad. I always tests shell script with bash.